src/Models/BaseElement.php

```diff
@@ -66,6 +68,12 @@ class BaseElement extends DataObject implements CMSPreviewable
      */
     private static $description = 'Base element class';
 
+    /**
+     * List of fields to exclude from CMS SiteTree seatch
+     * @see ElementSiteTreeFilterSearch::applyDefaultFilters()
+     */
+    private static array $fields_excluded_from_cms_search = [];
```

Comment:
```suggestion
    private static array $fields_excluded_from_cms_search = [
        'ExtraClass',
    ];
```
as per https://github.com/silverstripe/silverstripe-elemental/pull/1069#discussion_r1251298926

Reply:
Updated